Lib/asyncio/base_events.py

```diff
@@ -155,6 +156,63 @@ def _run_until_complete_cb(fut):
     futures._get_loop(fut).stop()
 
 
+
+class _SendfileProtocol(protocols.Protocol):
```

Comment:
Rename to `_SendfileFallbackProtocol`?

Reply:
Sorry, maybe I'm too tired but I want to ask explicitly: do you prefer java-stylish `VeryLongNameDesribingTheUniversityAndAllRelatedThings` names?
The `_SendfileProtocol` name is private. We can change it at any moment. User should never see/touch it. There is not name conflict, and the name is not cryptic abbreviation.
Why use longer version?

Reply:
I care about longer version because I want the code of asyncio to be easy to read.

I know about your work on this PR and we've discussed it many times privately, but still, it took me a while to read and understand the patch.  Here I was thinking that this protocol is used even for native-sendfile implementation.

`_SendfileProtocol` vs `_SendfileFallbackProtocol` is not a Java-like name. It's about as long as `_UnixSelectorEventLoop`. Feel free to grep Python source code and discover that there are plenty of way longer class names.

In this case, `_SendfileFallbackProtocol` disambiguates the purpose of this protocol, suggesting the reader what is the actual purpose of this piece of code.